CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -241,16 +241,21 @@ private void CloseImeComposition()
         private void OnImeComposition(IntPtr hwnd, int lParam)
         {
             string text = string.Empty;
-
+                
             if (ImeHandler.GetResult(hwnd, (uint)lParam, out text))
             {
                 owner.GetBrowserHost().ImeCommitText(text, new Range(int.MaxValue, int.MaxValue), 0);
+                if (languageCodeId == ImeNative.LANG_KOREAN)
+                {
+                    owner.GetBrowserHost().ImeSetComposition(text, new CompositionUnderline[0], new Range(int.MaxValue, int.MaxValue), new Range(0, 0));
+                    owner.GetBrowserHost().ImeFinishComposingText(false);
+                }
             }
             else
             {
                 var underlines = new List<CompositionUnderline>();
                 int compositionStart = 0;
-
+                
```

Comment:
Please revert formatting change please.

Reply:
Oh the whitespaces! I see. Sorry for the inconvenience. :-)